File: w1e08/w1e08.cs
using System;

namespace w1e08
{
    internal class Program
    {
        public static void Main(string[] args)
        {
            Console.ResetColor();
            int size = ReadInt("Please input size of tree");
            if (size < 2)
            {
                Console.Error.WriteLine("Sorry, the smalles I can draw is a size 2.");
                return;
            }

            for (int row = 0; row < size; row++)
            {
                int offset = size - row;
                if (offset > 0)
                {
                    Console.Write(new String(' ', size - row));
                }
                Console.BackgroundColor = ConsoleColor.Green;
                Console.ForegroundColor = ConsoleColor.Yellow;
                Console.Write(new String('*', 2 * row + 1));
                Console.ResetColor();
                Console.WriteLine();
            }
            Console.Write(new String(' ', size - 1));
            Console.BackgroundColor = ConsoleColor.DarkYellow; //Brown;
            Console.Write(new String("[ ]"));
            Console.ResetColor();
            Console.WriteLine();
        }

        private static int ReadInt(string question)
        {
            while (true)
            {
                Console.Write(question);
                Console.WriteLine(": ");
                try
                {
                    return Int32.Parse(Console.ReadLine() ?? throw new FormatException());
                }
                catch (FormatException)
                {
                    Console.Error.WriteLine("That's not a number, try again");
                }
            }
        }
    }
}

File: w1e08/w1e08.php
<?php
declare(strict_types=1);

use Codedungeon\PHPCliColors\Color;
use Puggan\Oop2019\Console;

require_once __DIR__ . '/../vendor/autoload.php';

Console::ResetColor();
$size = Console::ReadInt('Please input size of tree');

if ($size < 2)
{
    Console::ErrorWriteLine('Sorry, the smalles I can draw is a size 2.');
    exit();
}

foreach(range(0, $size) as $row)
{
    echo
        str_repeat(' ', $size - $row),
        Color::YELLOW, Color::BG_GREEN,
        str_repeat('*', 2 * $row + 1),
        Color::RESET,
        PHP_EOL;
}
echo str_repeat(' ', $size - 1), Color::BG_YELLOW, '[ ]', Color::RESET, PHP_EOL;
